src/app/modules/verification/verification.service.ts

```diff
@@ -159,10 +267,10 @@ export const getNewOtp = async (
   answer: string,
 ): Promise<void> => {
   // TODO (#317): remove usage of non-null assertion
-  if (isTransactionExpired(transaction.expireAt!)) {
+  if (isDateExpired(transaction.expireAt!)) {
```

Comment:
can we not use `isTransactionExpired` function exported from `verification.util`? Seems like the same thign

Reply:
yes it is! this is just a rename because the name conflicts with the new `isTransactionExpired`. I'm trying to isolate the changes to only `/:transactionId/reset` in this PR. `isDateExpired` gets replaced with `isTransactionExpired` in a subsequent PR.